packages/next/next-server/lib/fallback-context.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+
+export const FallbackContext = React.createContext<boolean>(
+  // This default is only used when a component does not have a matching
+  // Provider above it in the tree (e.g. test environment).
+  //
+  // We default to `true` because that would be the default rendered state
+  // in a non-browser scenario (SSR). Tests of components post-fallback should
+  // be using in-browser tests.
+  true
```

Comment:
We need to choose if we want this to default to `true` or `false` for Jest.

Reply:
I think `true` seems like an alright default, users could mock it during testing if they really wanted to change it to false right?

Reply:
Yeah they could always mock the context _or_ `useFallback`.